src/components/widgets/calender/clock.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+
+const Clock = React.createClass({
+ getInitialState(){
+  return(
+    {
+     hour: '',
+     minutes: '',
+     seconds: ''
+    }
+  )
+ },
+
+ componentDidMount(){
+  var today = new Date();
+  var hour = today.getHours();
+  var minutes = today.getMinutes();
+  var seconds = today.getSeconds();
+  var startTime = () =>{
+   for(var i = 0; i < 0; i++){
+    
+   
+   }
+  }
+   
+   this.setState({
+    hour: hour, minutes: minutes, seconds: seconds
+   })
```

Comment:
remove commented out code before opening a pull request. commented out code should not get merged into master